### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by KARTHIK07GUPTA**
author: Specifies the name of the package author.
author_email: Specifies the email address of the package author. install_requires: Specifies the dependencies required by the package. Add the necessary dependencies as a list of strings. entry_points: Allows you to define command-line scripts or other entry points for your package. It associates the sgm-cli command with the main() function in the sgm.cli module. package_data: Specifies additional files or data to be included with the package. In this example, it includes all text files in the data directory of the sgm package. license: Specifies the license under which the package is distributed (e.g., "MIT", "Apache License 2.0", etc.). classifiers: Provides classifiers or tags for the package that indicate its properties, target audience, and compatibility. Feel free to modify these fields according to your package's specific requirements.

### CODE REVIEW

**Review by brycedrennan — CHANGES_REQUESTED** [review]
Filled in values are wrong.

**Comment by airdrop11111** [comment]
I noticed a small bug in your code. Have you considered using XYZ to fix it?

**Comment by benjaminaubin** [comment]
The `setup.py` package will be updated with final values (authors, emails, license, requirements) for the public release date mid July

File: setup.py [hunk 1]
```diff
@@ -3,11 +3,33 @@
...
     description="Stability Generative Models",
     long_description=open("README.md", "r", encoding="utf-8").read(),
     long_description_content_type="text/markdown",
     url="https://github.com/Stability-AI/generative-models",
+    license="MIT",
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3",
```
Comment (ben-alkov) [inline setup.py:31]: Current README advises "This is tested under python3.8 and python3.10. For other python versions, you might encounter version conflicts."

**Pull request closed without merging**